packages/convenient_test_manager_dart/pubspec.yaml

```diff
@@ -17,10 +17,9 @@ dependencies:
   intl: ^0.17.0
   meta: ^1.3.0
   mobx: ^2.0.6
-  pedantic: ^1.11.0
   recase: ^4.0.0
   test_api: ^0.4.2
-  vm_service: ^8.0.0
+  vm_service: any
```

Comment:
Oops does not look like best practice... `>=8.0.0 <10.0.0' may be better if you need two major versions

Reply:
Oh, thanks, didn't know that I can do this in `dependencies` section as well :)